_includes/payment-order-other-features.md

```diff
@@ -541,7 +536,9 @@ for the given operation.
 
 ### View Payment Order
 
-The `view-paymentorder` operation contains the URI of the JavaScript that needs to be set as a `script` element's `src` attribute, either client-side through JavaScript or server-side in HTML as shown below.
+The `view-paymentorder` operation contains the URI of the JavaScript that needs
+to be set as a `script` element's `src` attribute, either client-side through
```

Comment:
Why the blank line? :)

Reply:
I just wrapped line 544 to 3 lines, it only looks like a blank line at line 539 because the red line 544 wraps to two virtual/visual (but not physical) lines.

Reply:
Oki doke!